app/main.py

```diff
@@ -2,0 +1,15 @@
+def copy_file(command: str) -> None:
+    command_words = command.split(" ")
+
+    if len(command_words) != 3:
+        return
+
+    command_name, file_in_path, file_out_path = command_words
+
+    if command_name == "cp" and (file_in_path != file_out_path):
+        with (
+            open(file_in_path, "r") as file_in,
+            open(file_out_path, "w") as file_out
+        ):
```

Comment:
It's a good practice to handle exceptions that may occur during file operations. Consider using a try-except block to catch potential errors such as FileNotFoundError or PermissionError, and provide appropriate feedback to the user.